domain/include/cstone/gravity/treewalk.hpp

```diff
@@ -112,7 +112,9 @@ void computeGravityGroup(TreeNodeIndex groupIdx,
             LocalParticleIndex numTargets  = lastTarget - firstTarget;
 
             // apply multipole to all particles in group
-            #pragma clang loop vectorize(enable)
+			#if defined(__llvm__) || defined(__clang__)
+				#pragma clang loop vectorize(enable)
+			#endif
```

Comment:
spurious tab characters? Please configure to IDE to only use space characters for indentation.

Reply:
You were right. I reinstalled the IDE eclipse some days ago. I forgot to change in the 'Formatted' tab by spaces. Now It is fix and commit.